interface/resources/qml/AssetServer.qml

```diff
@@ -311,6 +338,25 @@ Window {
                 colorScheme: root.colorScheme
                 anchors.left: parent.left
                 anchors.right: parent.right
+                rowDelegate: Rectangle {
+                    height: hifi.dimensions.tableRowHeight
+                    color: styleData.selected
+                        ? hifi.colors.primaryHighlight
+                        : treeView.isLightColorScheme
+                        ? (styleData.alternate ? hifi.colors.tableRowLightEven : hifi.colors.tableRowLightOdd)
+                    : (styleData.alternate ? hifi.colors.tableRowDarkEven : hifi.colors.tableRowDarkOdd)
```

Comment:
Removed this block, I had originally put the mousearea here but ended up moving it out